src/index.js

```diff
@@ -25,7 +27,7 @@ const mapRoutes = (routes, pathToController) => {
   const routesArr = entries(routes);
 
   routesArr.forEach((value) => {
-    let middlewares = [];
+    let middlewares = [middlewareGeneral];
```

Comment:
It would be advisable to allow adding more than one middleware to be added to the general route path.

Since `middlewareGeneral` is going to to be an array to allow more than one middlewares, it would be advisable that we spread it as shown below:

`let middlewares  = [ ...middlewareGeneral ];`